lisp/bazel-build.el

```diff
@@ -24,27 +24,42 @@
 
 ;;; Code:
 
+(defgroup bazel-build 'nil
+  "Package for editing, building, and running code using Bazel."
+  :link '(url-link "https://github.com/bazelbuild/emacs-bazel-mode")
```

Comment:
Maybe also add another link to https://bazel.build ?

Reply:
Sure, why not.